### ISSUE TITLE
check APERO  quality control varying with time

### ISSUE THREAD

**Issue by njcuk9999** [njcuk9999/apero-utils#62]
Many files have header keys starting with `QCC`

`QCC_ALL`   is 1 if all QC pass and 0 otherwise
`QCCXXXN` is the name of the quality control parameters
`QCCXXXV` is the measured value of that quality control parameter
`QCCXXXL` is the logic that was used (normally the failure criteria)
`QCCXXXP` is 1 if this QC passed and 0 otherwise

where XXX is a number from 000 to 999

e.g. a `waveref` file has 6 QC checks

```
QCC001N = 'MIN YWAVE DIFF A'   / Quality control parameter name
QCC001V =    6.520432253283616 / Quality control measured value
QCC001L = 'MIN YWAVE DIFF A < 0' / Quality control logic used
QCC001P =                    1 / Quality control param passed QC
QCC002N = 'MIN XWAVE DIFF A'   / Quality control parameter name
QCC002V = 0.002471618585900615 / Quality control measured value
QCC002L = 'MIN XWAVE DIFF A < 0' / Quality control logic used
QCC002P =                    1 / Quality control param passed QC
QCC003N = 'MIN YWAVE DIFF B'   / Quality control parameter name
QCC003V =    6.520143507880334 / Quality control measured value
QCC003L = 'MIN YWAVE DIFF B < 0' / Quality control logic used
QCC003P =                    1 / Quality control param passed QC
QCC004N = 'MIN XWAVE DIFF B'   / Quality control parameter name
QCC004V = 0.002472668112659449 / Quality control measured value
QCC004L = 'MIN XWAVE DIFF B < 0' / Quality control logic used
QCC004P =                    1 / Quality control param passed QC
QCC005N = 'DV[A - B]'          / Quality control parameter name
QCC005V =   0.3292754385959686 / Quality control measured value
QCC005L = 'abs(DV[A - B]) > 50000.0 m/s' / Quality control logic used
QCC005P =                    1 / Quality control param passed QC
QCC006N = 'CCFRV[A] - CCFRV[B]' / Quality control parameter name
QCC006V =   0.8588184165654287 / Quality control measured value
QCC006L = 'abs(CCFRV[A] - CCFRV[B]) > 50000.0 m/s' / Quality control logic used
QCC006P =                    1 / Quality control param passed QC
QCC_ALL =                    T

```

The idea is for each unique QCC (defined by QCCXXXN) that is numeric plot all files of this same suffix
(i.e. `_pp_e2dsff_B_wavesol_ref_B.fits`) and the limit against MJDMID (time)

Note:
- Some files on a single night will have the same QC do not duplicate these normally those with the same prefix (stuff before the `_pp`) will have the same QC but check this
- QC stands for "Quality control"
- Some QC checks will not be numerical - you cannot plot these

**Comment by fgenest1** [comment]
It seems that in pp files (just the *_pp.fits files in the tmp folders), the name and value of the QCCs are switched, i.e. QCCXXXN gives the value and QCCXXXV gives the name. This doesn't happen in any of the other file types I checked. For now, I'm putting a small workaround to account for that in my script.

**Comment by njcuk9999** [comment]
Weird, which values exactly (the XXX bit) this is a bug in apero that I need to fix.

**Comment by fgenest1** [comment]
All three QC checks for the pp.fits files have that issue, so QCC001N, QCC002N and QCC003N give values while QCC001V, QCC002V and QCC003V give names. I included a screenshot of part of a header as an example.
<img width="684" alt="Screenshot 2023-06-15 at 4 32 40 PM" src="https://github.com/njcuk9999/apero-utils/assets/131305570/ddb60d27-a856-4fe8-962c-a19983f48280">

**Comment by njcuk9999** [comment]
Okay I found the typo this should be fixed in v0.8.274+, I'm not sure how you want to handle having both cases (though there should be a version key in the header)
